var/spack/repos/builtin/packages/atlas/package.py

```diff
@@ -107,28 +107,23 @@ def install(self, spec, prefix):
             make("install")
             self.install_test()
 
-    def setup_dependent_package(self, module, dspec):
-        # libsatlas.[so,dylib,dll ] contains all serial APIs (serial lapack,
-        # serial BLAS), and all ATLAS symbols needed to support them. Whereas
-        # libtatlas.[so,dylib,dll ] is parallel (multithreaded) version.
-        name = 'libsatlas.%s' % dso_suffix
-        libdir = find_library_path(name,
-                                   self.prefix.lib64,
-                                   self.prefix.lib)
-
-        if '+shared' in self.spec:
-            self.spec.blas_shared_lib   = join_path(libdir, name)
-            self.spec.lapack_shared_lib = self.spec.blas_shared_lib
+    @property
+    def blas_libs(self):
+        shared = True if '+shared' in self.spec else False
+        return find_libraries(['libsatlas'], root=self.prefix, shared=shared, recurse=True)
```

Comment:
that will not work, unfortunately, as there is no static libs `libsatlas.a`. I would do the following:

```
    variant('openmp', default=False,
            description="Use multithreaded libraries.")
```
and
```
if '+shared' in self.spec:
     to_find = ['libtatlas'] if '+openmp` in self.spec else ['libsatlas']
     shared = True
else:
     # TODO: different libs for multithreaded
     to_find = ['cblas' 'f77blas' 'atlas']
     shared = False
```